contrib/sssd.spec.in

```diff
@@ -1050,6 +1047,13 @@ getent passwd sssd >/dev/null || useradd -r -g sssd -d / -s /sbin/nologin -c "Us
 %systemd_post sssd-pam.socket
 %systemd_post sssd-ssh.socket
 %systemd_post sssd-sudo.socket
+%__rm -f %{mcpath}/passwd
+%__rm -f %{mcpath}/group
+%__rm -f %{mcpath}/initgroups
+%__rm -f %{mcpath}/sid
+%__chown %{sssd_user}:%{sssd_user} %{dbpath}/*
+%__chown %{sssd_user}:%{sssd_user} %{_sysconfdir}/sssd/sssd.conf
+%__chown -R %{sssd_user}:%{sssd_user} %{_sysconfdir}/sssd/conf.d/*
```

Comment:
Hi,

I got the error
```
/usr/bin/chown: cannot access '/etc/sssd/conf.d/*': No such file or directory
warning: %post(sssd-common-9.pr7193-04224.fc39.x86_64) scriptlet failed, exit status 1
```
Using `-f` will suppress the error message but the command will still return `1'. Since you are using `-R` and afaik we do not support sub-directories in `conf.d` I wonder if you meant
```
%__chown -R %{sssd_user}:%{sssd_user} %{_sysconfdir}/sssd/conf.d
```
which would chown the `conf.d` directory itself as well.

bye,
Sumit